src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -228,9 +244,13 @@ class DropdownAutoCompleteMenu extends React.Component {
             <List
               width={width}
               style={{outline: 'none'}}
-              height={Math.min(items.length * virtualizedHeight, maxHeight)}
+              height={this.getHeight(items)}
```

Comment:
Perhaps we should keep this part and still respect the `maxHeight`?
e.g. `Math.min(this.getHeight(items), maxHeight)`

Reply:
I moved this to `getHeight`, but it is still there. Do you think it would be clearer if I did something like `height={Math.min(this.getMinHeight(), maxHeight)}`?

Reply:
Ahh no sorry, I somehow didn't see that it was there earlier. All good.